docs/hardware/adding.md

```diff
@@ -192,13 +192,13 @@ to respective `ACCELEROMETER_*` lines as defined in the bootloader.
 If possible, keep this separate from the SDA/SCL exposed on the header,
 so the one on the header can be used as a general digital IO.
 
-## Vibration motor
+## Vibration motor #vibrationmotor
```

Comment:
why are all the above all 3rd level under Schematics but the vibration motor and the others below are 2nd level, on a par with Schematics?

Reply:
Reorganization the component sections